docs/examples/batch_request.md

```diff
@@ -96,7 +96,136 @@ My next 2 events are House Warming Party,Some Foo Event.
 
 It should also contain a response containing two photos from the user.
 
-> **Warning:** The response object should return a `null` response for any request that was pointed to with JSONPath as is [the behaviour of the batch functionality of the Graph API](https://developers.facebook.com/docs/graph-api/making-multiple-requests/#operations).
+> **Warning:** The response object should return a `null` response for any request that was pointed to with JSONPath as is [the behaviour of the batch functionality of the Graph API](https://developers.facebook.com/docs/graph-api/making-multiple-requests/#operations). If we want to receive the response anyway we have to set the `omit_response_on_success` option to `false`. [See the example below](#force-response-example).
+
+## Force Response Example
+
+The following example is a subset of the [first example](#example). We will only use the `user-events` and `post-to-feed` requests of the [first example](#example), but in this case we will force the server to return the response of the `user-events` request.
+
+```php
+$fb = new Facebook\Facebook([
+  'app_id' => '{app-id}',
```

Comment:
The indentation here and in the other PHP code examples in this file seems to be off, can we consistently use PSR-2 here instead and indent with 4 spaces?